### PR TITLE
fix: add missing edges to filter

### PR DESCRIPTION
**Pull request by JonasBK**
## Description

Add four ADCS edges to path finding filter selection.

## How Has This Been Tested?

Edges are filtered out if they are deselected as expected.

## Screenshots (if appropriate):

image

## Types of changes

-   [ ] Chore (a change that does not modify the application functionality)
-   [X] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [ ] Documentation updates are needed, and have been made accordingly.
-   [ ] I have added and/or updated tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] My changes include a database migration.

### CODE REVIEW

File: cmd/ui/src/views/Explore/ExploreSearch/edgeTypes.tsx [hunk 1]
```diff
@@ -89,13 +89,17 @@ export const AllEdgeTypes: Category[] = [
             {
                 name: 'Active Directory Certificate Services',
                 edgeTypes: [
-                    ActiveDirectoryRelationshipKind.RootCAFor,
-                    ActiveDirectoryRelationshipKind.PublishedTo,
-                    ActiveDirectoryRelationshipKind.ManageCertificates,
-                    ActiveDirectoryRelationshipKind.ManageCA,
                     ActiveDirectoryRelationshipKind.DelegatedEnrollmentAgent,
                     ActiveDirectoryRelationshipKind.Enroll,
```
Comment (urangel) [inline cmd/ui/src/views/Explore/ExploreSearch/edgeTypes.tsx:93]: I believe we only list transitive edges in this filter so the enroll edge, for instance, would not be something that should be populated here. You'll notice getChanges and getChangesAll are absent from any of the AD filters.

**Comment by juggernot325** [comment]
The failing tests here should be fixed by #187. I just merged that into `adcs-poc`, so merging that into your branch should resolve this.

File: cmd/ui/src/views/Explore/ExploreSearch/edgeTypes.tsx (hunk 1 above)
Comment (JonasBK) [reply cmd/ui/src/views/Explore/ExploreSearch/edgeTypes.tsx:93]: Oh yeah, you are right.. Good catch. It is actually only GoldenCert, and the ESCX edges that are transitive. So maybe we should remove all the others. I will ask Andy if he agrees

Comment (urangel) [reply cmd/ui/src/views/Explore/ExploreSearch/edgeTypes.tsx:93]: Yea that sounds correct to only include GoldenCert and the ESCX edges. Thanks for confirming with Andy

**Review by rvazarkar — APPROVED** [review]

**Pull request merged**